https://bugs.gentoo.org/508748

fix building w/freetype-2.5.3

patch by Jungshik Shin <jshin@chromium.org>

--- Imaging-1.1.7/_imagingft.c
+++ Imaging-1.1.7/_imagingft.c
@@ -70,7 +70,11 @@
     const char* message;
 } ft_errors[] =
 
+#if defined(USE_FREETYPE_2_1)
+#include FT_ERRORS_H
+#else
 #include <freetype/fterrors.h>
+#endif
 
 /* -------------------------------------------------------------------- */
 /* font objects */
